src/main/java/com/google/opensesame/servlets/PersonEntity.java

```diff
@@ -0,0 +1,28 @@
+package com.google.opensesame.servlets;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import java.util.ArrayList;
+
+@Entity
+public class PersonEntity {
+  @Id String id;
+  ArrayList<String> interestTags;
+  String email;
+  String gitHubId;
+
+  public PersonEntity() {}
+
+  public PersonEntity(PersonObject person) {
+    this.gitHubId = person.getGitHubID();
+    this.interestTags = person.getTags();
+    this.email = person.getEmail();
+  }
+
+  public PersonEntity(String id, String gitHubId, ArrayList<String> interestTags, String email) {
+    this.id = id;
+    this.gitHubId = gitHubId;
+    this.interestTags = interestTags;
+    this.email = email;
+  }
```

Comment:
Can we deprecate PersonObject and just use `PersonEntity` instead now? Even better, could we just modify `PersonObject` slightly so that it works with Objectify? It seems like both the objects store the same data, so I feel like one may be enough.

Reply:
Hi Joe! So one of the main reasons we still have Objects and Entities is due to the GitHub API call limit. We don't want to store all of the parameters we get from GitHub (eg. name, location, bio, profile pic) in datastore, so we want to keep the Entities as light as possible. However, we also don't want to have to query GitHub every time we want one of these parameters. Obi and I thought that, by turning a light Entity into a Robust object once using the GitHub API, we will be able to have direct access to all parameters without having to make repeated API calls.